tests/e2e/builder/cases/lodash/index.swc.test.ts

```diff
@@ -30,15 +30,15 @@ test('should optimize lodash bundle size when using SWC plugin', async () => {
   expect(size < 10).toBeTruthy();
 });
 
-test('should not optimize lodash bundle size when transformLodash is false and using SWC plugin', async () => {
+// TODO: fixed in rsbuild next version, see https://github.com/web-infra-dev/rsbuild/pull/1572
```

Comment:
we can release Rsbuild 0.4.3 to fix this today.

Reply:
ok, waiting for 0.4.3

Reply:
released: https://github.com/web-infra-dev/rsbuild/releases/tag/v0.4.3

Reply:
updated